wikipendium/wiki/views.py

```diff
@@ -107,7 +109,15 @@ def add_tag_to_article(request, slug):
     if 'tag' not in request.POST:
         return HttpResponseBadRequest()
 
-    tag = slugify(request.POST['tag'])
+    slug = original_slug = slugify(request.POST['tag'])
```

Comment:
We still refer to the original tag when storing the name here: https://github.com/stianjensen/wikipendium.no/pull/420/files#diff-948051c43b1f2e9080e28689ea48fd93R116